schemas/stsci.edu/asdf/0.1.0/core/table.yaml

```diff
@@ -0,0 +1,108 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/0.1.0/core/table"
+tag: "tag:stsci.edu:asdf/0.1.0/core/table"
+
+title: >
+  A table.
+
+description: |
+  A table is represented as a list of columns, where each entry is a
+  [column](ref:http://stsci.edu/schemas/asdf/0.1.0/core/column)
+  object, containing the data and some additional information.
+
+  The data itself may be stored inline as text, or in binary in either
+  row- or column-major order by use of the `strides` property on the
+  individual column arrays.
+
+  Each column in the table must have the same first (slowest moving)
+  dimension.
+
+examples:
+  -
+    - A table stored in column-major order, with each column in a separate block
+    - |
+        !core/table
+          columns:
+          - !core/column
+            data: !core/ndarray
+              source: 0
+              datatype: float64
+              byteorder: little
+              shape: [3]
+            description: RA
```

Comment:
Not a big deal, but might be nice to have the example(s) be a bit more realistic by having column names `ra`, `dec`, `id` and descriptions `RA (J2000)`, `Dec (J2000)`, `Source identifier`.